README.md

```diff
@@ -462,11 +462,17 @@ Events:
 
 #### byzantine_bad_chain
 
+These should be treated as a prompt to mass exit immediately.
+
 Events:
 
 **invalid_block**
 
-Event informing about that particular block is invalid.
+Event informing about that particular block is invalid
+
+**unchallenged_exit**
```

Comment:
1. Don't you think this event will be thrown too often(lots of false alarms)? 

I think there will be more valid/proper exits than invalid ones so from users perspective there is no point to challenged valid exit if the proof doesn't exist. 

Am I missing something?

Reply:
> active exit having gone too long without being challenged

so this only is raised if the unchallenged exit approaches finality (which is very bad). This isn't about "currently isn't challenged" but "hasn't been challenged".

ah I see what's missing, the word "invalid", thx good catch